open-api/rest-catalog-open-api.yaml

```diff
@@ -3142,6 +3163,10 @@ components:
           type: object
           additionalProperties:
             type: string
+        storage-credentials:
+          type: array
+          items:
+            $ref: '#/components/schemas/Credential'
```

Comment:
nit: should this object be called `StorageCredential` in this case?

Reply:
renamed